### PR TITLE
♻️ Conform Tabs tokens

### LINKED ISSUE #1254: Conform Tabs tokens

### PR DESCRIPTION
**Pull request by mimarz**
resolves #1254

### CODE REVIEW

File: libraries/core-react/src/components/Tabs/Tabs.tokens.ts [hunk 1]
```diff
@@ -15,67 +16,96 @@ const {
       primary__hover: { hex: activeHoverColor },
```
Comment (pomfrida) [inline libraries/core-react/src/components/Tabs/Tabs.tokens.ts:16]: Should change hex to rgba

**Review by pomfrida — CHANGES_REQUESTED** [review]
Nice rewrite 👍  Just change hex to rgba in tokens then it's good to go

File: libraries/core-react/src/components/Tabs/Tabs.tokens.ts (hunk 1 above)
Comment (mimarz) [reply libraries/core-react/src/components/Tabs/Tabs.tokens.ts]: Fixed

**Review by pomfrida — APPROVED** [review]
LGTM 👍

**Pull request merged**